tools/labs/templates/kernel_api/6-list-sync/list-sync.c

```diff
@@ -24,7 +24,7 @@ struct task_info {
 
 static struct list_head head;
 
-/* TODO: you can use either a spinlock or rwlock, define it here */
+/* TODO 1: you can use either a spinlock or rwlock, define it here */
 DEFINE_RWLOCK(lock);
```

Comment:
@teoserbanescu careful here. Please also check that the output skeleton looks fine. 

AFAIK a syntax like:
```
TODO 1/3

foo1();
foo2();
foo3();
```

removes first 3 lines after TODO from output skeleton.

I'm not sure what ```TODO 1``` does. Maybe is as expected. But please check the output.

Reply:
It was behaving as expected, removing only one line. I still modified this in another commit to be more clear.

Reply:
@dbaluta the number of lines is 1 by default if you skip it.